kuma/search/filters.py

```diff
@@ -184,7 +188,8 @@ def filter_queryset(self, request, queryset, view):
             if len(filter_tags) > 1:
                 facet_params = F('terms', tags=list(filter_tags))
             else:
-                facet_params = F('term', tags=filter_tags[0])
+                if filter_tags:
+                    facet_params = F('term', tags=filter_tags[0])
```

Comment:
This will still break the code when ``filter_tags`` is empty, because ``facet_params`` will be undefined and the next line that uses ``facet_params`` will fail.